src/util.cpp

```diff
@@ -42,7 +42,7 @@
 #include <llvm/Support/FileSystem.h>
 #include <llvm/Support/Path.h>
 
-#ifdef ISPC_HOST_IS_APPLE
+#if defined(ISPC_HOST_IS_APPLE) || defined(ISPC_HOST_IS_LINUX)
```

Comment:
Should it be the check for `libc++`? I.e. `#ifdef ( _LIBCPP_VERSION)`.

Reply:
It is a good point. Fixed.